model/gen-ai/spans.yaml

```diff
@@ -71,6 +71,18 @@ groups:
         requirement_level: recommended
       - ref: gen_ai.usage.output_tokens
         requirement_level: recommended
+      - ref: gen_ai.system.instructions
+        requirement_level: opt_in
+      - ref: gen_ai.input.messages
```

Comment:
can we define some limit on the instrumentation side before SDK supports it

Reply:
added a note that instrumentations can truncate, let's consider adding limits as a follow up